src/Entity/Document.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace Pucene\Bundle\DoctrineBundle\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+
+/**
+ * Document entity.
+ */
+class Document
```

Comment:
This classes should be placed in the library and in the bundle only the doctrin-xml should be placed.

what do you think about this idea?